mobile/src/main/java/org/openhab/habdroid/ui/AbstractBaseActivity.kt

```diff
@@ -170,6 +173,21 @@ abstract class AbstractBaseActivity : AppCompatActivity(), CoroutineScope {
         }
     }
 
+    @CallSuper
+    override fun onPrepareOptionsMenu(menu: Menu): Boolean {
+        // TODO: Use correct color
+        val iconColor = resolveThemedColor(R.attr.colorPrimary)
+        menu.forEach { item ->
+            val icon = item.icon
+            icon?.let {
+                icon.mutate()
+                DrawableCompat.setTint(icon, iconColor)
+                item.icon = icon
+            }
+        }
+        return super.onPrepareOptionsMenu(menu)
+    }
```

Comment:
Can't this be done in XML? https://stackoverflow.com/a/54961263

Reply:
I didn't know the "app" prefix and tried "android" instead, which isn't backwards compatible.